src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
-
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball firstBall = Lottery.getRandomBall();
+        System.out.println(firstBall);
+        Ball secondBall = Lottery.getRandomBall();
+        System.out.println(secondBall);
+        Ball thirdBall = Lottery.getRandomBall();
+        System.out.println(thirdBall);
     }
```

Comment:
Use a for loop for creating several objects of the same class. Instead of manually creating three `Ball` instances, you can use a loop to create them. This will reduce code duplication and make your code easier to maintain.